### PR TITLE
Adds sticky nav for small screens only

### PR DESCRIPTION
**Pull request by jarmstrng**
- Adds sticky nav on mobile with hamburger menu and Register button
- Removes hamburger menu from desktop/tablet
- Hides intro section Register button on mobile
- Adds footer height to compensate for nav height

### CODE REVIEW

File: assets/css/custom.css [hunk 1]
```diff
@@ -110,24 +112,110 @@ body, #qcSiteNav h4, #qcPriNav ul > li > a, #qcPriNav ul ul > li > a, .dash_titl
...
 	letter-spacing: 2px;
-}
+}*/
 
-a:not(.button):hover {
+/*a:not(.button):hover {
 	color: #172751;
 	cursor: pointer;
-}
+}*/
```
Comment (deweydell) [inline assets/css/custom.css]: If you add `.button` class to the register now button, you won't have to comment out my link styles. 
Sorry if I didn't explain this when I wrote it, but this is how I like to have a default link style with designs that display `a` elements as both links and buttons. And then we can build on the `.button` class if we want to expand button styles. 

Does that make sense? Let me know if you'd prefer it another way!

Comment (deweydell) [reply assets/css/custom.css]: So like for example a regular hyperlink can just be an `a` element. Where as a link disguised as a button will be `<a class="button prospectus-link"/>` or `<a class="button register-now"`

**Comment by deweydell** [comment]
Other than my button comment, I checked on my Pixel and it looks good to me :)

File: assets/css/custom.css (hunk 1 above)
Comment (jarmstrng) [reply assets/css/custom.css]: ok cool! i did mean to ask you about the button and then simply forgot in my rush to get this deployed before leaving the office. i'll definitely give your suggestion a try :)

**Pull request merged**